### PR TITLE
Remove pagingToken from getEvents

### PR DESCRIPTION
**Pull request by psheth9**
Delete `pagingToken` from getEvents. Use `Cursor` instead. 

Note: This is a breaking change !! 

Fixes: https://github.com/stellar/soroban-rpc/issues/118

### CODE REVIEW

**Comment by mollykarcher** [comment]
Shouldn't this be targeted to `v22-breaking-changes`?

File: Makefile [hunk 1]
```diff
@@ -77,11 +81,19 @@ clean:
...
 build-soroban-rpc: build-libs
-	go build -ldflags="${GOLDFLAGS}" ${MACOS_MIN_VER} -o soroban-rpc -trimpath -v ./cmd/soroban-rpc
+	go build -ldflags="${GOLDFLAGS}" ${MACOS_MIN_VER} -o ${SOROBAN_RPC_BINARY} -trimpath -v ./cmd/soroban-rpc
+
+# the build-stellar-rpc build target is an optimized build target used by
+# https://github.com/stellar/pipelines/blob/master/soroban-rpc/Jenkinsfile-soroban-rpc-package-builder
+# as part of the package building.
+build-stellar-rpc: build-libs
+	go build -ldflags="${GOLDFLAGS}" ${MACOS_MIN_VER} -o ${STELLAR_RPC_BINARY} -trimpath -v ./cmd/soroban-rpc
+
```
Comment (2opremio) [inline Makefile:96]: All these changes seem unrelated to the PR

Comment (psheth9) [reply Makefile:96]: landed those changes to `main` yesterday, that's why it is showing up in diff !! `v22-breaking-change` need to pull new changes from main

**Comment by Shaptic** [comment]
@2opremio @mollykarcher maybe I missed this but why do we have a distinction between v22-breaking-changes and protocol-22? I feel like everything in `protocol-22` is breaking?

**Comment by mollykarcher** [comment]
honestly i don't think it was intentional? we definitely only need one

File: Makefile (hunk 1 above)
Comment (psheth9) [reply Makefile:96]: @2opremio can you merge `main` to `v22-breaking-change` so that I can close this PR 😄

Comment (psheth9) [reply Makefile]: ^^ This is fixed now !!

**Review by aditya1702 — APPROVED** [review]
